app/controllers/tag_controller.rb

```diff
@@ -41,13 +41,18 @@ def show
                         .page(params[:page])
                         .order("node_revisions.timestamp DESC")
     end
-
+    respond_with(@nodes) do |format|
```

Comment:
This looks good, but I think this has to go down on line 51, at the end of the action method, or some of the variables used in the templates won't exist when we try to render it. Make sense? Thanks!